zerver/lib/user_groups.py

```diff
@@ -268,7 +274,8 @@ def create_system_user_groups_for_realm(realm: Realm) -> Dict[int, UserGroup]:
         everyone_on_internet_system_group,
     ]
 
-    UserGroup.objects.bulk_create(system_user_groups_list)
+    system_groups_to_create = [*system_user_groups_list, nobody_system_group]
```

Comment:
I just want to mention if we can add `nobody_system_group` at the top of the `system_user_groups_list` and skip `nobody_system_group` to create `GroupGroupMembership` by 
````
subgroup, remaining_groups = system_user_groups_list[1], system_user_groups_list[2:]
````